tox.ini

```diff
@@ -36,15 +36,17 @@ envdir = .tox/docs
 deps =
   -r requirements-dev.txt
   sphinx-intl
-commands = sphinx-build -b html {posargs} {toxinidir}/docs {toxinidir}/docs/_build/html
+  qiskit-aer
```

Comment:
An alternative to setting `qiskit-aer` as a requirement when building the docs would be to set the environment variable `QISKIT_SUPPRESS_PACKAGING_WARNINGS=Y` under `[testenv:docs]` (i.e.):

```
setenv =
  QISKIT_SUPPRESS_PACKAGING_WARNINGS=Y
```

Setting the environment variable would suppress the warnings `Could not import Aer provider...` and not require installing `qiskit-aer` when building the docs.

Reply:
How long does the doc build take if `qiskit-aer` is included?

Reply:
When running locally, I don't notice much of a difference. The average time it takes to build, whether aqua is included or not, is about 78 seconds. When aqua is included, it takes about an extra second to build lol.